File: Java/Leetcode/TwoPointers/ReverseString.java
package com.brianprojects.Leetcode.TwoPointers;

public class ReverseString {
    public void reverseString(char[] s){
        if(s.length <0 || s == null){
            return;
        }

        for(var i = 0; i<s.length/2; i++){
            var temp = s[i];
            s[i] = s[s.length-1-i];
            s[s.length-1-i] = temp;
        }
    }
}

File: Python/Leetcode_Python/TwoPointers/ReverseString.py
from typing import List


def reverseString(s: List[str]) -> None:
    if not s or len(s) == 0:
        return

    p1 = 0
    p2 = len(s) - 1

    while p1 < p2:
        temp = s[p1]
        s[p1] = s[p2]
        s[p2] = temp
        p1 += 1
        p2 -= 1
